src/main/java/transcation/TransactionList.java

```diff
@@ -0,0 +1,31 @@
+package transcation;
+
+import java.util.ArrayList;
+
+public class TransactionList {
+    public static final String REMOVE_TRANSACTION_FORMAT = "remove-tx /t [TRANSACTION_ID]";
+
+    private static final ArrayList<Transaction> transactionList = new ArrayList<>();
+    public static void removeTransaction(String userInput) {
+
+        String[] words = userInput.split(" ");
+        if (
+                (words.length < 2) || (!words[1].equals("/t"))
```

Comment:
I think it's better if the 'if' condition is inline with the 'if' keyword.

Move this line (condition) up to the if statement.